resources/js/components/table.js

```diff
@@ -1,90 +1,91 @@
-import $ from 'jquery'
-import 'datatables.net-buttons/js/buttons.html5.min'
+import DataTable from 'datatables.net-dt';
+import {wprtSendApiRequest} from "./api-helper";
```

Comment:
@damladag I think we didn't have that apiRequest imported here, so if it is not needed please remove this.